apps/api/src/app/database/entity/project.entity.ts

```diff
@@ -40,6 +40,9 @@ export class ProjectEntity {
     @Column({ length: 32 })
     public network: string
 
+    @Column("text", { default: "a future project" })
```

Comment:
Empty string default is fine

Reply:
Yea, filler should be handled on the front-end